lexicons/tools/ozone/moderation/defs.json

```diff
@@ -131,6 +131,21 @@
           "format": "datetime",
           "description": "Timestamp referencing the first moderation status impacting event was emitted on the subject"
         },
+        "recordUpdatedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Last update timestamp of the record the subject is associated with"
+        },
+        "recordDeletedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp referencing when the record the subject is associated with was deleted"
+        },
+        "recordStatus": {
+          "type": "string",
+          "description": "Status of the record the subject is associated with. Statuses are different when the subject references an account vs. a record",
+          "knownValues": ["takendown", "suspended", "deleted", "deactivated"]
+        },
```

Comment:
I guess I was using `record` as a generic term here without the context of atproto records. I'm also not sure about `subject` prefix either since we already have `updatedAt` column on subject, we will end up with `subject.updatedAt` and `subject.subjectUpdatedAt` which feels odd. I think I'm onboard with the term `hosting` just from technical standpoint.